### PR TITLE
Feature/issue 11

### PR DESCRIPTION
**Pull request by NervzZ**

### CODE REVIEW

**Review by edouardmichelin — CHANGES_REQUESTED** [review]
Move all common components to `presentation/ui/common`.
You can also group them by type, e,g.:
- form composables in `common/form` (`EmailTextField`, `NumericTextField`)
- text composables in `common/text` (`BigLabel`, `BasicTextField`)

Don't forget the newline at end of file :)

File: app/src/androidTest/java/com/studhub/app/ui/BasicFilledButtonTest.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    fun componentBasicFilledButtonExistsWithCorrectGivenNameWhenCreated() {
+        composeTestRule.setContent {
+            BasicFilledButton(onClickHandler = { /*TODO*/ }, label = "Test")
+        }
+
+        composeTestRule.onNodeWithText("Test").assertExists()
+    }
+
+    @Test
+    fun componentBasicFilledButtonCorrectlyInvokePassedMethodWhenClicked() {
```
[the remark this answers is not in the document]
Comment (edouardmichelin) [reply app/src/androidTest/java/com/studhub/app/ui/BasicFilledButtonTest.kt:28]: See `ListingUseCaseTest` functions

[the remark this answers is not in the document]
Comment (edouardmichelin) [reply app/src/androidTest/java/com/studhub/app/ui/BasicFilledButtonTest.kt:28]: Actually doesn't work in `androidTest`

[the remark this answers is not in the document]
Comment (NervzZ) [reply app/src/androidTest/java/com/studhub/app/ui/BasicFilledButtonTest.kt:28]: I wish that was true for android tests

File: app/src/main/java/com/studhub/app/RegisterUserActivity.kt [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                        phoneNumber = phoneNumber.value,
+                        firstName = firstName.value,
+                        lastName = lastName.value,
+                        userName = userName.value,
+                        profilePicture = "pf_placeholder.png"
+                    )
+                    scope.launch {
+                        submit(user)
+                    }
+                },
```
[the remark this answers is not in the document]
Comment (NervzZ) [reply app/src/main/java/com/studhub/app/RegisterUserActivity.kt:123]: Only usable within composable context, annoyingly. Moved it up with other variable declarations

**Review by Edouard-6 — APPROVED** [review]
Add newlines at end of files, otherwise LGTM

**Review by edouardmichelin — APPROVED** [review]
LGTM

**Pull request merged**